static/forums/templates/user-login.html

```diff
@@ -26,15 +32,17 @@ <h4>Please login with your <br>Spoken-Tutorial.org account.</h4>
                     <input type="hidden" name="next" value="{{ next }}">
                 {% endif %}
                 <input class="btn btn-sm btn-primary" type="submit" value="Login">
+		<span style="margin-left:30px;"><a href="http://spoken-tutorial.org/accounts/forgot-password/?next=http://forums.spoken-tutorial.org/accounts/login/?resetpass=success">Forgot Password?</a></span>
```

Comment:
why does it have `resetpass=success`?
Also, use django `{% url '' %}` template tag instead of hard coding the url.